test/integration/deep/norelay/testdata/client.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: client
+spec:
+  selector:
+    matchLabels:
+      app: client
+  template:
+    metadata:
+      labels:
+        app: client
+    spec:
+      containers:
+      - name: curl
+        image: curlimages/curl
+        command: [ "sh", "-c", "--" ]
+        args: [ "while true; do sleep 10; done;" ]
```

Comment:
Would this still work without this sleep loop?

Reply:
No, the container will finish and k8s will continuously restart it.